jumpscale/sals/process/__init__.py

```diff
@@ -569,8 +569,7 @@ def is_port_listenting(port):
     Returns:
         Bool: True if port is used else False
     """
-    pcons = [proc for proc in psutil.net_connections() if proc.laddr.port == port and proc.status == "LISTEN"]
```

Comment:
I didn't see this, but this would just break it on other platforms.

Reply:
Will revert until we find another solution that's fit for both.

Reply:
I bypassed the error on the line you see with a hack to get to the other error, also, why would ignoring the FileNotFound Error break the routine on other platforms? For me it only gave an error on iTerm2

Reply:
there is an issue for the  error on the line you marked  as well: https://github.com/threefoldtech/js-sdk/issues/215

Reply:
damned, I indeed deleted that line to bypass an error there

Reply:
should I make a pull request just for https://github.com/threefoldtech/js-sdk/issues/216 ?

Reply:
oh, sure!
please do.

Reply:
https://github.com/threefoldtech/js-ng/pull/415